File: PHP/uberX.php
<?php
require_once('car.php');

class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license,$driver,$brand,$model){
        parent::_construct($license,$driver);
        $this -> license = $license;
        $this -> driver = $driver;
    }
}

File: Python/uberx.py
from car import Car

class UberX(Car):
    brand = str;
    model = str;
    
    #Aplicando herencia de Car
    def __init__(self,license,driver,brand,model):
        super().__init__(license,driver)
        self.brand = brand
        self.model = model
        